client/src/components/nav_comp/DemoLoginButton.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { login } from '../../store/auth';
+import { Redirect } from 'react-router-dom';
+import '../../style/loginmodal.css'
+
+function LoginButton(){
+    const username = "yusuke@example.com"
+    const password = "password"
+    const currentUserId = useSelector(state => state.auth.id);
+    const dispatch = useDispatch();
+    
+    const handleSubmit = (e) => {
+        e.preventDefault();
+        dispatch(login(username, password))
+    }
```

Comment:
I would suggest moving the `username` and `password` into the `handleSubmit` function since you aren't using the variables anywhere outside of the function.
```suggestion
    const currentUserId = useSelector(state => state.auth.id);
    const dispatch = useDispatch();
    
    const handleSubmit = (e) => {
        e.preventDefault();
        const username = "yusuke@example.com";
        const password = "password";
        dispatch(login(username, password))
    }
```